packages/react-strict-dom/src/native/modules/createStrictDOMComponent.js

```diff
@@ -638,6 +639,14 @@ export function createStrictDOMComponent<T, P: StrictProps>(
         if (nativeComponent === Pressable) {
           nativeComponent = AnimatedPressable;
         }
+      } else {
+        // shim for `transform` property was done with the animated property in the if condition
+        // above, but if we have reached till here, it means that we are not yet able to shim `transform`
+        // property and going to check it here now.
+        const transformValue = styleProps.style.transform;
+        if (transformValue != null && typeof transformValue === 'string') {
+          styleProps.style.transform = parseTransform(transformValue);
+        }
```

Comment:
I don't understand what this comment is saying, but since transform always needs to be parsed now, let's do it in the `stylex` shim instead of here.

Reply:
I initially thought to keep it one place and add the `transform` shim only in `stylex`. If I remove the code from here and do everything in stylex, then I would have to also move the logic of `transitionProperty` and `transform` parsing from (useStyleTransition)[https://github.com/facebook/react-strict-dom/blob/c10e6a95385612e868b283b4e905e0ec94289f7d/packages/react-strict-dom/src/native/modules/useStyleTransition.js#L139] to stylex too. Is that okay?